rosdep/python.yaml

```diff
@@ -380,6 +380,10 @@ pyros-setup-pip:
   ubuntu:
     pip:
       packages: [pyros-setup]
+pyrr-pip:
```

Comment:
Can you please make this key follow the guidance here: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1 ?